Shared/Presenter/BaseItemListPresenter.swift

```diff
@@ -64,25 +67,26 @@ class BaseItemListPresenter {
     }()
 
     lazy private(set) var cancelObserver: AnyObserver<Void> = {
-        return Binder(self) { target, filterText in
+        return Binder(self) { target, _ in
             target.dispatcher.dispatch(action: ItemListFilterAction(filteringText: ""))
             target.dispatcher.dispatch(action: ItemListFilterEditAction(editing: false))
         }.asObserver()
     }()
 
-    lazy fileprivate var emptyPlaceholderItems = [
-        ItemSectionModel(model: 0, items: [LoginListCellConfiguration.EmptyListPlaceholder(learnMoreObserver: self.learnMoreObserver)]
-        )
-    ]
+    lazy private(set) var retryNetworkObserver: AnyObserver<Void> = {
+        return Binder(self) { target, _ in
+            target.dispatcher.dispatch(action: NetworkAction.retry)
+            }
+            .asObserver()
+    }()
 
-    lazy fileprivate var noResultsPlaceholderItems = [
-        ItemSectionModel(model: 0, items: [LoginListCellConfiguration.NoResults(learnMoreObserver: self.learnMoreNewEntriesObserver)]
-        )
-    ]
+    lazy fileprivate var emptyPlaceholderItems = [LoginListCellConfiguration.EmptyListPlaceholder(learnMoreObserver: self.learnMoreObserver)]
+
+    lazy fileprivate var noResultsPlaceholderItems = [LoginListCellConfiguration.NoResults(learnMoreObserver: self.learnMoreNewEntriesObserver)]
+
+    lazy internal var syncPlaceholderItems = [LoginListCellConfiguration.SyncListPlaceholder]
 
-    lazy internal var syncPlaceholderItems = [
-        ItemSectionModel(model: 0, items: [LoginListCellConfiguration.SyncListPlaceholder])
-    ]
+    lazy internal var noNetworkItems = [LoginListCellConfiguration.NoNetwork(retryObserver: self.retryNetworkObserver)]
```

Comment:
Does this mean that when the network isn't available, then neither are your passwords? That doesn't seem very useful.

Reply:
no, you'll still see the items show up in _addition_ to the no network disclaimer item.